queries.sql

```diff
@@ -22,6 +25,38 @@ BEGIN;
 delete from animals;
 ROLLBACK;
 select * from animals;
+=======
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* Query to update and read the animals table data */
+BEGIN;
+SELECT species FROM animals; -- verify original values
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals; -- verify changes made
+ROLLBACK;
+SELECT species FROM animals; -- verify that change was rolled back
+COMMIT;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT species from animals; -- verify that change was made
+
+COMMIT;
+SELECT species from animals; -- verify that change persists after commit
+
+BEGIN;
+DELETE FROM animals WHERE species = 'unspecified';
+SELECT COUNT(*) FROM animals; -- verify that the delete occurred
+
+ROLLBACK;
+SELECT COUNT(*) FROM animals; -- verify that the rollback happened
```

Comment:
- [ ] according to the requirements you should Write queries to answer the following:

    Who was the last animal seen by William Tatcher?
    How many different animals did Stephanie Mendez see?
    List all vets and their specialties, including vets with no specialties.
    List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
    What animal has the most visits to vets?
    Who was Maisy Smith's first visit?
    Details for most recent visit: animal information, vet information, and date of visit.
    How many visits were with a vet that did not specialize in that animal's species?
    What specialty should Maisy Smith consider getting? Look for the species she gets the most.